gradle/defaults-java.gradle

```diff
@@ -6,6 +6,8 @@ allprojects {
     targetCompatibility = "11"
 
     compileJava.options.encoding = "UTF-8"
+    compileJava.options.compilerArgs << '-Xlint:unchecked'
```

Comment:
I added linting options in a separate commit.

Reply:
Yep, saw that and I'll update this to master and resolve.